aleph_proxy_config/main.py

```diff
@@ -26,13 +27,23 @@
 
 async def download_nodes() -> Dict:
     # Iterate over trusted hosts in case the first is unavailable
+    last_error = None
     for trusted_host in TRUSTED_HOSTS:
         url = trusted_host + PATH
-        async with aiohttp.ClientSession(timeout=aiohttp.ClientTimeout(30)) as session:
-            async with session.get(url) as response:
-                response.raise_for_status()
-                data = await response.json()
-                return data
+        try:
+            async with aiohttp.ClientSession(timeout=aiohttp.ClientTimeout(30)) as session:
+                async with session.get(url) as response:
+                    response.raise_for_status()
+                    data = await response.json()
+                    return data
+        except ClientResponseError as error:
+            logger.exception(f"Error downloading nodes from {trusted_host}")
+            last_error = error
+            continue
+    else:
```

Comment:
`else` is unnecessary, you will always enter this bit of code.

Reply:
Yes, but then you won't be in the function anymore so the `else` is still unnecessary.